### PR TITLE
(nightfall-browser) web-worker comlink implement to fetch circuits files from s3 bucket

### LINKED ISSUE #469: Retrieve relevant proof generation inputs to wallet
Definition of Done:

### PR DESCRIPTION
**Pull request by LijuJoseJJ**
this worker thread fetch/download all files in s3 bucket and insert it in IndexedDB.
fixes #469

### CODE REVIEW

File: hosted-utils-api-server/src/routes/browser-circuit.mjs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+import AWS from 'aws-sdk';
+import config from 'config';
+
+const { accessKeyId, secretAccessKey } = config.AWS;
+AWS.config.update({ accessKeyId, secretAccessKey });
+const s3 = new AWS.S3();
+
+const myBucket = 'nightfallv3';
```
Comment (IlyasRidhuan) [inline hosted-utils-api-server/src/routes/browser-circuit.mjs:9]: separate out the buckets from which to get the circuits + abis and the proving keys. Make them environmental variables as the stub and non-stub varieties might be stored elsewhere

**Comment by Westlad** [comment]
Can you add some instructions for testing too please?

File: hosted-utils-api-server/src/routes/browser-circuit.mjs (hunk 1 above)
Comment (LijuJoseJJ) [reply hosted-utils-api-server/src/routes/browser-circuit.mjs]: done

**Comment by LijuJoseJJ** [comment]
hi @Westlad 
so to test this PR, if we can run browser e2e-test locally(successfully), without doing old manual downloading/copying circuits file from AWS s3 bucket `aws s3 cp s3://nightfallv3/circuits ./wallet/src/zokrates --recursive` and then running script `.wallet/copy-zokrates-local.sh` to copy local proving key to `./wallet/src/zokrates`. 

Previously we were doing manual download and copy of the file (and storing it in .gitignore file `./wallet/src/zokrates`). Now, this PR does automatically at start of page load in worker thread and then store as unit8Array in indexedDB 
Screenshot 2022-03-22 at 11 11 00 AM

also, in this PR I have added instruction (`./wallet/README.md`) to run bowser e2e test

**Review by ChaitanyaKonda — APPROVED** [review]

**Pull request merged**